File: 1512-number-of-good-pairs/1512-number-of-good-pairs.c
int numIdenticalPairs(int* nums, int numsSize) {
    int result = 0;
    for(int i = 0; i < numsSize; i++) {
        for(int j = i + 1; j < numsSize; j++) {
            if (nums[i] == nums[j]) result++;
        }
    }    
    return result;
}

File: 1512-number-of-good-pairs/1512-number-of-good-pairs.js
var numIdenticalPairs = function(nums) {
    let mp = new Map();
    let result = 0;
    for(let i = 0; i < nums.length; i++) {
        result += mp.get(nums[i]) || 0;
        mp.set(nums[i], (mp.get(nums[i]) || 0) + 1);
    }
    return result;
};